core/templates/dev/head/components/CollectionCreationService.js

```diff
@@ -54,7 +54,7 @@ oppia.factory('CollectionCreationService', [
           }, function() {
             $rootScope.loadingMessage = '';
           }
-        );
+          );
```

Comment:
This seems weird. Why is it indented at this level?

EDIT: Ah, it's because that's where the left paren opens. In this case move this to the end of the previous line.